src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomIndex = new Random().nextInt(Colors.values().length);
```

Comment:
create class variable and use it

Reply:
You mean create a variable in Color Supplier class and use in method?
But I don't understand which variable I have to put in a Color Supplier class, because in fact I generate a random index and just take the value from the enum.